Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -9401,6 +9401,228 @@
+        <Collide index="Ground" value="1"/>
+        <Collide index="Structure" value="1"/>
+        <Collide index="RoachBurrow" value="1"/>
+        <Collide index="ForceField" value="1"/>
+        <Collide index="Locust" value="1"/>
+        <Collide index="Small" value="1"/>
+        <Attributes index="Armored" value="1"/>
+        <Attributes index="Biological" value="1"/>
+        <Attributes index="Structure" value="1"/>
+        <LifeStart value="1250"/>
+        <LifeMax value="1250"/>
+        <LifeArmor value="1"/>
+        <LifeRegenRate value="0.2734"/>
+        <LifeArmorName value="Unit/LifeArmorName/ZergBuildingArmor"/>
+        <StationaryTurningRate value="719.4726"/>
+        <TurningRate value="719.4726"/>
+        <Sight value="10"/>
+        <CostCategory value="Technology"/>
+        <CostResource index="Minerals" value="150"/>
+        <CostResource index="Vespene" value="75"/>
+        <AttackTargetPriority value="11"/>
+        <AbilArray Link="BuildinProgressNydusCanal"/>
+        <AbilArray Link="AP_OmegaCreepBuild"/>
+        <AbilArray Link="AP_OmegaTrain"/>
+        <AbilArray Link="Rally"/>
+        <AbilArray Link="AP_que8"/>
+        <AbilArray Link="AP_SalvageNydus"/>
+        <AbilArray Link="AP_OmegaTrainSwarmling"/>
+        <BehaviorArray Link="AP_makeCreep8x6"/>
+        <CardLayouts>
+            <LayoutButtons Face="AP_BuildCreepTumorPropagate" Type="AbilCmd" AbilCmd="AP_OmegaCreepBuild,Build1" Row="2" Column="0"/>
+            <LayoutButtons Face="AP_NydusWormSubterraneanScales" Type="Passive" Requirements="AP_HaveNydusWormSubterraneanScales" Row="1" Column="0"/>
+            <LayoutButtons Face="AP_BirthRaptor" Type="AbilCmd" AbilCmd="AP_OmegaTrainSwarmling,Train1" Row="2" Column="1"/>
+            <LayoutButtons Face="AP_BirthRaptor" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train3" Row="2" Column="1"/>
+            <LayoutButtons Face="AP_BirthSwarmling" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train2" Row="2" Column="1"/>
+            <LayoutButtons Face="AP_BirthZergling" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train1" Row="2" Column="1"/>
+            <LayoutButtons Face="AP_BirthCorpser" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train6" Row="2" Column="2"/>
+            <LayoutButtons Face="AP_BirthVile" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train7" Row="2" Column="2"/>
+            <LayoutButtons Face="AP_BirthRoach" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train5" Row="2" Column="2"/>
+            <LayoutButtons Face="AP_BirthHydraliskImpaler" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train9" Row="2" Column="3"/>
+            <LayoutButtons Face="AP_BirthHydraliskLurker" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train10" Row="2" Column="3"/>
+            <LayoutButtons Face="AP_BirthHydralisk" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train8" Row="2" Column="3"/>
+            <LayoutButtons Face="AP_BirthAberration" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train11" Row="2" Column="4"/>

```

Comment:
You're missing the button for cancel (like Barracks has). I'd move all train buttons to the first row (that's currently empty) thus creating a room at bottom right for cancel
![Snímek obrazovky pořízený 2024-08-02 16-09-02](https://github.com/user-attachments/assets/c74a9664-05ae-4cb7-871c-128786f39fbb)
![Snímek obrazovky pořízený 2024-08-02 16-08-33](https://github.com/user-attachments/assets/7408338b-bcf4-476f-98af-ce5c7d8a0a7a)

Reply:
Also you can queue 8 lings but the 8th is hidden as the bottom row of queues can display only up to 6 elements